config/requirements.txt

```diff
@@ -18,5 +18,6 @@ django-ordered-model~=3.1.1
 django-redis~=4.10.0
 django-debug-toolbar~=1.11
 django-cacheops~=4.1
+git+https://github.com/garyburgmann/drf-firebase-auth.git # Needed until next release
```

Comment:
Can the bot handle an update or we should update it manually?

Reply:
I don't think the bot has an option to do that, but I'm watching for new releases on their GitHub so we can do it manually